### PR TITLE
Updated Load-generator.md

### PR DESCRIPTION
**Pull request by Rahulkumar2002**
In load-generator.md updated GetNightHawk to NightHawk

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit b5c1f0f8f1d925946d725e3bc30d321cdf56276b at: https://6181419fab599fecd032de7b--meshery-docs-preview.netlify.app

**Review by leecalcote — COMMENTED** [review]
Thanks, @Rahulkumar2002. Will you also update the hyperlink in the line above to be https://getnighthawk.dev?

**Comment by Rahulkumar2002** [comment]
> Thanks, @Rahulkumar2002. Will you also update the hyperlink in the line above to be https://getnighthawk.dev?

Ok I will try , Can you point out the repo for this hyperlink ?

File: docs/pages/extensibility/load-generators.md [hunk 1]
```diff
@@ -24,4 +24,4 @@ Different use cases and different opinions call for different approaches to stat
 1. [fortio](https://github.com/fortio/fortio) - Fortio load testing library, command line tool, advanced echo server and web UI in go (golang). Allows to specify a set query-per-second load and record latency histograms and other useful stats.
 1. [wrk2](https://github.com/giltene/wrk2) - A constant throughput, correct latency recording variant of wrk.
 1. [nighthawk](https://github.com/envoyproxy/nighthawk) - Enables users to run distributed performance tests to better mimic real-world, distributed systems scenarios.
```
Comment (warunicorn19) [inline docs/pages/extensibility/load-generators.md:26]:
```suggestion
1. [nighthawk](https://getnighthawk.dev/) - Enables users to run distributed performance tests to better mimic real-world, distributed systems scenarios.
```
@Rahulkumar2002, @leecalcote is referring to this.

**Review by warunicorn19 — CHANGES_REQUESTED** [review]

**Review by leecalcote — APPROVED** [review]

**Pull request merged**